lib/domain/SyntaxSearch.js

```diff
@@ -151,4 +159,16 @@ export default class SyntaxSearch extends SyntaxModel {
             }
         }
     }
+
+    onRequestError(error) {
+        if (this.searchView) {
+            if (error.code === 'ENOTFOUND') {
```

Comment:
For requests that call this handler on error, test for:
- if 'ENOTFOUND', expect that No internet connection reported
- For all other errors, expect that it shows error connecting to SyntaxDB